config-linux.md

```diff
@@ -494,6 +494,98 @@ You MUST specify at least one of the `hcaHandles` or `hcaObjects` in a given ent
 }
 ```
 
+## <a name="configLinuxUnified" />Unified
+
+**`unified`** (object, OPTIONAL) allows cgroup v2 parameters to be modified at runtime for the container.
+
+Each key in the map refers to a file in the cgroup unified hierarchy.
+
+The OCI runtime MUST ensure that the needed cgroup controllers are enabled for the cgroup.
+
+Configuration unknown to the runtime MUST still be written to the relevant file.
```

Comment:
I am not sure why we need this. Did you have a use case in mind?

Reply:
it was pointed out here: https://github.com/opencontainers/runtime-spec/pull/1040#discussion_r435832813

When the runtime finds a configuration like `"$CONTROLLER.$NAME": "$VALUE"`, it must always try to write it to the `$CONTROLLER.$NAME` file in the hierarchy.  This is mostly needed to prevent that the OCI runtime ignores any of the cgroup v2 settings.